### ISSUE TITLE
Separate phase analyses of one skill

### ISSUE THREAD

**Issue by HannahWyatt68** [0todd0000/spm1dmatlab#84]
Hi Todd,

We're currently looking at the sidestepping skill in two distinct phases (preparatory and stance) and are planning on running SPM analyses for each phase in isolation (0-100% for each). I understand that this approach may be considered artificial smoothing of the data and will likely increase the fwhm value and lower the critical t-value. Would it therefore be considered to be a breach of SPM assumptions? If so, is there anything that can be done to overcome this and still be able to analyse the phases in isolation? 

Many thanks in advance,

Hannah

**Comment by 0todd0000** [comment]
Hi Hannah,

If the two distinct phases have noticeably different smoothness, I can see why it could be considered artificial smoothing to look at them separately, and I agree that it would be, but only if one's *a priori* hypothesis pertains to the whole continuum.  Let's consider two *a priori* hypothesis cases:

* **Case 1**:  The *a priori* hypothesis pertains to the whole continuum (i.e. to effects in general without specific reference to the phases). In this case the phases should not be separated because individual phases are not directly relevant to the hypothesis.

* **Case 2**:  Two *a priori* hypotheses exist and pertain explicitly to the two phases.  In this case the phases should be separated, and a correction for the two (multiple) comparisons should be made to retain the  alpha rate across both comparisons.

Regarding SPM assumptions and their possible breach:
The short answer is that both Case 1 and Case 2 are valid, and do not violate SPM assumptions, even if the two phases have noticeably different smoothness.

This issue is referred to as "anisotropic" or "non-isotropic" smoothness in the SPM literature, and refers to the fact that different parts of the continuum can naturally embody different frequencies. On the surface it seems like anisotropic could pose a problem for inference, but it actually does not.  The reason is that SPM's smoothness estimation procedures implicitly correct for anisotropic smoothness, and that this implicit correction is equivalent to an explicit correction.  First let's consider the explicit correction:

An explicit correction for anisotropic smoothness is provided in Worsley et al. (1999). From the perspective of 1D continuum analysis, the correction is as follows:  if the continuum contains *Q* nodes (lets say *Q*=101), then an explicit correction for anisotropic smoothness proceeds by first estimating local smoothness, which can be done using the gradient at each node. For all continuum measurements, even ones with apparently uniform smoothness, the numerical gradient is generally non-constant at each node. The second step is to calculate the number of nodes *Q*' for which the continuum has constant smoothness. This effectively stretches out the continuum (let's say to *Q*'=130.4 nodes), where the continuum now has constant smoothness (constant gradient) at every node, and the smoothness of this new continuum dataset is equivalent to the smoothest part of the original data. Then inference is conducted using the new continuum, which has *Q*' nodes.

[Worsley, K., Andermann, M., Koulis, T., MacDonald, D., & Evans, A. (1999). Detecting changes in nonisotropic images. Human Brain Mapping, 8(2-3), 98–101.](https://doi.org/10.1002/(SICI)1097-0193(1999)8:2/3<98::AID-HBM5>3.0.CO;2-F)

However, this explicit correction is unnecessary, because SPM's implicit correction accomplishes the same thing.  The implicit correction, and the one that SPM procedures use by default, is to simply use the average gradient across the continuum. In other words, conducting inference using *Q* nodes and the average smoothness is equivalent to using *Q*' nodes and the greatest smoothness.

To summarize:

* If one part of the continuum is rougher than another part, it may seem that one should correct for this, because the rougher part is more likely to produce high statistical values simply by chance.

* An explicit correction for anisotropic smoothness is unnecessary when the *a priori* hypothesis pertains to the whole continuum because SPM's smoothness estimation procedures implicitly correct for anisotropic smoothness.

* Phases should be separately analyzed only when one has *a priori* hypotheses that pertain to the separate phases.

Todd

**Comment by HannahWyatt68** [comment]
Hi Todd,

Thank you for the helpful reply and thorough explanation - it's very much appreciated!

Kind regards,

Hannah

**Comment by 0todd0000** [comment]
Hi again Hannah,

I'm writing with a brief follow-up in case you are in search of additional references regarding this issue. The paper below was recently accepted for publication, and is available in [the paper's GitHub repository](https://github.com/0todd0000/nonuniform1d) as "manuscript.pdf". The paper numerically explores a variety of issues relevant to non-constant smoothness. The main conclusions are that:

1. SPM thresholds (random field theory thresholds) are valid for nonuniform smoothness, provided the data are normally distributed.

2. SPM's cluster-specific p values may be slightly inaccurate when smoothness is non-constant, but since cluster-specific p values are less than alpha (by definition), these inaccuracies are negligible for classical hypothesis testing purposes.


> Pataky TC, Vanrenterghem J, Robinson MA, Liebl D (2019). On the validity of statistical parametric mapping for nonuniformly and heterogeneously smooth one-dimensional biomechanical data. Journal of Biomechanics, in press.

Todd
